contracts/Content/UniqueContent.sol

```diff
@@ -23,18 +33,13 @@ contract UniqueContent is IUniqueContent, IMultipleRoyalties, MultipleRoyalties,
     function mint(LibAsset.UniqueAssetCreateData memory _data) external override {
         require((IERC1155Upgradeable(_data.contentAddress).balanceOf(_msgSender(), _data.tokenId)) >= 1, "You must have the original item in your wallet");
```

Comment:
Add a check here first if the contentAddress supports IERC1155 and IERC2981. You'll have to fail if it doesn't support ERC1155 or ERC2981

Reply:
Failing Early here allows us to avoid checking for ERC1155 and ERC2981 later on in other functions or in the storage contract